modin/engines/base/io/file_dispatcher.py

```diff
@@ -51,9 +54,28 @@ def _read(cls, *args, **kwargs):
         raise NotImplementedError(NOT_IMPLEMENTED_MESSAGE)
 
     @classmethod
-    def get_path(cls, file_path):
+    def get_path(cls, file_path: str, glob_path: bool = False) -> Union[list, str]:
```

Comment:
I think I prefer a different utility for globs here.

Reply:
fixed.